package.json

```diff
@@ -2,8 +2,10 @@
   "version": "1.0.0",
   "scripts": {
     "start": "vite",
+    "start:ipfs": "HASH_ROUTING=true vite --base ./",
     "serve": "vite preview",
-    "build": "scripts/build",
+    "build": "vite build",
+    "build:ipfs": "HASH_ROUTING=true vite build --base ./",
```

Comment:
Why do we need to supply `--base ./` here?

Reply:
Because for example with `https://ipfs.fleek.co/ipfs/QmYkhVxzVpidFbXoaGCiu1WxVA6nmzGcFnbnUKuTG7benT/#/` we need to specify the `base` as `./` so we can use a relative path for the assets.
Else we would try to fetch all the assets from `https://ipfs.fleek.co/` which wouldn't find them.

Will try to deploy an example which shows the issue at hand

Reply:
All good, just wanted to know.